bson error

Hi,
I'm trying to run the project and getting this error : 
{ [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
js-bson: Failed to load c++ bson extension, using pure JS version

I found a solution that looks good here: http://stackoverflow.com/questions/28651028/cannot-find-module-build-release-bson-code-module-not-found-js-bson

but i'm new to Docker and didn't understand how I can see all the project's files, I can't see where you are using this module. 
any ideas how to solve the issue? 
thanks

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 
 # ---
 
@@ -21,7 +21,6 @@ RUN cd /tmp && npm install
 ADD . /app
 RUN rm -rf /app/node_modules
 RUN cp -a /tmp/node_modules /app/
-
 # ---
 
 EXPOSE 49090

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"dependencies": {
-		"mongoose": "^3.8.14",
+		"mongoose": "~4.4",
 		"express": "^4.8.2",
 		"body-parser": "^1.6.2",
 		"jade": "^1.5.0"

